### PR TITLE
Finished initial instructions 

### PR DESCRIPTION
**Pull request by cryptokoala4**
Updated git README.md and added description and url

### CODE REVIEW

File: Planning.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+MBTI: Which Game of Thrones character are you?
```
Comment (f3r) [inline Planning.txt]: Add a header by putting `# MBTI:...`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
```
Comment (f3r) [inline index.html]: extra spaces!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	<title>MBTI</title>
+    <script src="js/jquery.min.js"></script>
```
Comment (f3r) [inline index.html]: scripts should be just before `</body>`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <script src="js/script.js"></script>
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+</head>
+<body>
+  <h1 class="text-center">MBTI: Which Game of Thrones character are you?</h1>
+    <section class="text-center">
+    To see which Game of Thrones character you most match up with, try this game, a totally inaccurate variant on the Myers-Briggs style personality test.
+    </section>
+      <br>
```
Comment (f3r) [inline index.html]: indentation, `<br>` should be at the same level as `</section>`

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  var goQuiz = function() {
+    $('#start-quiz').click(function(){
+     window.location = "./quiz.html";
+    })
+  }
+
+  goQuiz();
+
+})
+
```
Comment (f3r) [inline js/script.js]: extra lines at the end of the file

File: README.md [hunk 6]
```diff
@@ -1 +1,3 @@
-# MBTI-GoT
+# MBTI - Which Game of Thrones characer you are
+
+#### Based on the Myers Briggs Type Indicator test, check out which GoT character you'd likely to be in George R.R. Martin's fantasy world through a series of depressing questions. There'll also be an analysis of your personality!
```
Comment (f3r) [inline README.md]: This should be more like:

```
### Description
Based on the Myers Briggs Type Indicator test, check out which GoT character you'd likely to be in George R.R. Martin's fantasy world through a series of depressing questions. 

There'll also be an analysis of your personality!
```

File: index.html [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+</head>
+<body>
+  <h1 class="text-center">MBTI: Which Game of Thrones character are you?</h1>
+    <section class="text-center">
+    To see which Game of Thrones character you most match up with, try this game, a totally inaccurate variant on the Myers-Briggs style personality test.
+    </section>
+    <br>
+      <article class="text-center">This is a popular way of judging personality types, assigning one of two traits each in four different categories to your personality: Introversion (preferring solitude and small groups of people) or Extroversion (preferring socializing); iNtuition (gathering information through the mental and abstract) or Sensing (gathering information through the concrete and physical); Feeling (making decisions based on empathy and emotion) or Thinking (making decisions based on detached rationality); and Perception (showing the outside world your intuition or sensing side) or Judgment (showing the outside world your feeling or thinking side). Thus, there are 16 personality types, such as ISTJ.
```
Comment (f3r) [inline index.html]: fix indentation

File: js/script.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+$(function(){
+
+  var goQuiz = function() {
+    $('#start-quiz').click(function(){
+     window.location = "./quiz.html";
```
Comment (f3r) [inline js/script.js]: fix indentation

File: quiz.html [hunk 9]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Quiz</title>
+    <script src="js/jquery.min.js"></script>
```
Comment (f3r) [inline quiz.html:6]: scripts before `</body>`

File: quiz.html [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+    <script src="js/bootstrap.min.js"></script>
+    <script src="js/script.js"></script>
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+</head>
+<body>
+  <form class="container-fluid col-">
```
Comment (f3r) [inline quiz.html]:
``` html
<div class="container-fluid">
  <div class="row">
    <form>
      ...
    </form>
  </div>
</div>
```

**Comment by f3r** [comment]
Please review comments before I can merge

**Comment by cryptokoala4** [comment]
Uh oh, hold on left out something

**Pull request closed without merging**